src/Raven.Studio/wwwroot/Content/scss/pages/_splashscreen.scss

```diff
@@ -0,0 +1,185 @@
+$orbit-base-size: 120px;
```

Comment:
I wonder if we should nest those styles to avoid clashes.

Reply:
The styles are now nested under `.splash-screen` class